src/odemis/util/driver.py

```diff
@@ -172,14 +172,15 @@ def estimateMoveDuration(distance, speed, accel):
         return t1 + t2
 
 
-def isNearPosition(current_pos, target_position, axes,
+def isNearPosition(current_pos, target_position, axes, rot_axes: set = None,
                    atol_linear: float = None,
                    atol_rotation: float = None):
     """
     Check whether given axis is near stage target position
     :param current_pos: (dict) current position dict (axis -> value)
     :param target_position: (dict) target position dict (axis -> value)
     :param axes: (set) axes to compare values
+    :param rot_axes: (set) rotational axes in the given set of axes (optional)
```

Comment:
Please add a note that if it's not provided, the axes which have a name starting with "r" in the `axes` param are considered rotational.